__tests__/app.test.js

```diff
@@ -191,29 +191,66 @@ describe("/api", () => {
-  //     test("200: responds with array of review objects", () => {
-  //       return request(app)
-  //         .get("/api/reviews")
-  //         .expect(200)
-  //         .then(({ body: { reviews } }) => {
-  //           expect(reviews.length).toBe(12);
-  //           expect(
-  //             reviews.forEach((review) => {
-  //               expect.objectContaining({
-  //                 owner: expect.any(String),
-  //                 title: expect.any(String),
-  //                 review_id: expect.any(Number),
-  //                 category: expect.any(String),
-  //                 review_img_url: expect.any(String),
-  //                 created_at: expect.any(String),
-  //                 votes: expect.any(Number),
-  //                 designer: expect.any(String),
-  //                 comment_count: expect.any(String),
-  //               });
-  //             })
-  //           );
-  //         });
-  //     });
-  //   });
+  describe("GET /api/reviews", () => {
+    test("200: responds with array of review objects sorted by date", () => {
+      return request(app)
+        .get("/api/reviews")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews).toBeSortedBy("created_at", { descending: true });
+          expect(reviews.length).toBe(13);
+          expect(
+            reviews.forEach((review) => {
+              expect(review).toEqual(
+                expect.objectContaining({
+                  owner: expect.any(String),
+                  title: expect.any(String),
+                  review_id: expect.any(Number),
+                  category: expect.any(String),
+                  review_img_url: expect.any(String),
+                  created_at: expect.any(String),
+                  votes: expect.any(Number),
+                  designer: expect.any(String),
+                  comment_count: expect.any(Number),
+                })
+              );
+            })
+          );
+        });
+    });
+    test("200: should accept a category query to filter reviews by category", () => {
+      return request(app)
+        .get("/api/reviews?category=social+deduction")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews.length).toBe(11);
+          expect(
+            reviews.forEach((review) => {
+              expect(review).toEqual(
+                expect.objectContaining({
+                  category: "social deduction",
+                })
+              );
+            })
+          );
+        });
+    });
+    describe("Errors", () => {
+      test("400: responds with error when category of wrong type", () => {
+        return request(app)
+          .get("/api/reviews?category=6")
+          .expect(400)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Incorrect datatype for category");
+          });
+      });
+      test('404: responds with error when there are no reviews with that category', () => {
```

Comment:
Have a think about how you would differentiate between a category that doesn't exist vs a category that exist but has no reviews (have a look in the test data for an example!)